sphinx/builders/latex/__init__.py

```diff
@@ -144,7 +144,7 @@ def get_outdated_docs(self):
     def get_target_uri(self, docname, typ=None):
         # type: (str, str) -> str
         if docname not in self.docnames:
-            raise NoUri
+            raise NoUri(docname)
```

Comment:
`typ` should be also captured.

Reply:
Are you sure? It's not used in the function so it does not influence the exception being raised or not (yet I don't know what contains `typ`).

Reply:
This is an usage of `typ`.
https://github.com/sphinx-doc/sphinx/blob/a6a2a59ab0abfaf510d3dcbc43231468d3dc1fff/sphinx/builders/manpage.py#L54-L55

I understand it is very less used. But it has been actually used. So no reason to ignore it.